src/fw/drivers/nrf5/qspi.c

```diff
@@ -29,9 +30,7 @@
 
 #define FLASH_RESET_WORD_VALUE (0xffffffff)
 
-static void prv_use(QSPIPort *dev) { dev->state->use_count++; }
-
-static void prv_release(QSPIPort *dev) { PBL_ASSERTN(dev->state->use_count > 0); }
```

Comment:
Ha, removing this is definitely correct... especially because we then would have crashed after 2^31 flash ops, since I apparently removed the decrement here!